psbt_channel_funder.go

```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+const (
+	// CustomChannelRemoteReserve is the custom channel minimum remote
+	// reserve that we'll use for our channels.
+	CustomChannelRemoteReserve = 1062
```

Comment:
Oddly precise number?

Reply:
Yeah, not sure where exactly this is coming from. Took over the commit from https://github.com/lightninglabs/taproot-assets/pull/934, @GeorgeTsagk did some calculations to arrive at it.
Probably the 1% of our 100k default channel size plus some dust margin?